packages/common/core-interfaces/src/events/listeners.ts

```diff
@@ -6,6 +6,9 @@
 /**
  * `true` iff the given type is an acceptable shape for a {@link Listeners | event} listener
  * @public
+ *
+ * @remarks
+ * Deprecated in `@fluidframework/tree`. Consider importing from `fluid-framework` or`@fluidframework/core-interfaces`  instead.
```

Comment:
Since the `@deprecation` in tree appears to be working, we can remove all of these comments, right?

Then you can also remove `"@fluidframework/core-interfaces": minor` from change set.